database/select_example.py

```diff
@@ -0,0 +1,13 @@
+from sqlalchemy import select
+from database import *
+from model import *
+
+stmt = select(Report).where(Report.id == '1')
+
+session = SessionLocal()
+
+result = session.execute(stmt)
+for report_obj in result.scalars():
+    print(report_obj.status.meaning)
+
+session.close()

```

Comment:
w sumie pytanie czy jeśli wyżej wystąpi błąd to sesja się zamknie? Często close używa się w bloku `finally` w połączniu z `try...except`.

Reply:
Wstawiony ww. blok.